src/ResourceManager/Sql/Commands.Sql/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 
 ## Version 4.11.5
 * Fixed issue where some backup cmdlets would not recognize the current azure subscription
+* Add DnsZonePartner Parameter for New-AzureRmSqlManagedInstance cmdlet to support AutoDr for Managed Instance.
```

Comment:
@ziwa-msft this should go under the `## Current Release` header, but we can move it when we do the versioning for the next release of this preview module